TGC Starter Guide.md

```diff
@@ -0,0 +1,93 @@
+\-\-- Setting up the Game \-\--
```

Comment:
do we need this document? It seems like we should try to just put things in the rules and make it simple there 😛